app/src/main/java/org/wikipedia/ktx/Context.kt

```diff
@@ -0,0 +1,12 @@
+@file:JvmName("ContextUtil")
+
+package org.wikipedia.ktx
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import androidx.core.content.getSystemService
+
+fun Context.setPlainText(label: CharSequence?, text: CharSequence?) {
```

Comment:
`setPlainText()` does not make sense as an extension function of `Context`.  Can we keep the file structure the way it was?

Reply:
Sure.